documentation/configuration/parameters/skipExecutingMigrations.md

```diff
@@ -10,13 +10,13 @@ redirect_from: /documentation/configuration/skipExecutingMigrations/
 {% include teams.html %}
 
 ## Description
-Whether Flyway should skip actually executing the contents of the migrations and only update the schema history table.
-    
-This should be used when you have applied a migration manually (via executing the sql yourself, or via an ide), and just want the schema history table to reflect this.
+Whether Flyway should skip migration execution. The remainder of the operation will run as normal - including updating the schema history table, callbacks, and so on.
 
-This is useful when executing an script against your database manually. For instance, when you apply an out-of-process change like a hotfix. The hotfix script can be turned into a migration and applied with `skipExecutingMigrations=true`. The schema history table will be updated, but the hotfix script won't be run gain.
+`skipExecutingMigrations` essentially allows you to 'pretend' that a migration was executed, because the schema history table is still updated as normal.
```

Comment:
I feel like if one needs quotes, then there's probably a better wording... how about "...allows you to mimic a migration being executed..."